discovery-provider/src/queries/queries.py

```diff
@@ -2251,3 +2251,20 @@ def get_previously_private_playlist():
         playlist_ids = [result[0] for result in previously_private_results]
 
     return api_helpers.success_response({ 'ids': playlist_ids })
+
+# Get the users for a given creator node ID
+@bp.route("/creator_node/users/<int:sp_node_id>", methods=("GET",))
+def get_creator_node_users(sp_node_id):
+    db = get_db_read_replica()
+    with db.scoped_session() as session:
+        user_query = (
+            session.query(User)
+                .filter(
+                    or_(
+                        and_(User.is_current == True, User.primary == sp_node_id),
+                        and_(User.is_current == True, User.secondaries.any(sp_node_id))
+                    )
+            ).all()
```

Comment:
nit - `iscurrent == true && (primary == spid || secondaries.any(spid)` is cleaner

Reply:
i think using the sqlalchemy features is actually cleaner than this proposed solution, re-open if you feel strongly but i did change it from when this was reviewd eliminating duplication of is_current == true